src/Sulu/Bundle/WebsiteBundle/Resources/public/js/services/reference-store.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
+
+    'use strict';
+
+    var synonyms = {},
+        references = {},
+
+        resolveAlias = function(alias) {
```

Comment:
the synonym can be configured by the bundle any other alias will fail. In sulu we have quite consistent names. but the SuluArticleBundle uses `articles` and `articles_page_tree` as smart-content provider alias which would not match without this synonyms.

On the other hand we spoken about multiple aliases where we would need such a feature.